pom.xml

```diff
@@ -295,6 +295,18 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <!-- TODO: This can be deleted after Spark 1.5 is posted -->
```

Comment:
All this looks fine; does this particular change belong only in the 1.5 branch though?